test/core_language/rebar.config

```diff
@@ -5,6 +5,10 @@
     {test, [
         {pre_hooks, [{compile, "cargo run -- build ."}]},
         {src_dirs, ["src", "test", "gen/src", "gen/test"]}
+    ]},
+    {gh_actions, [
```

Comment:
What's this new profile for?

Reply:
It's for running tests with prebuilt binary `run: rebar3 as gh_actions eunit` couldn't figure out how to achieve that with `cargo run -- build .` which was building a new one instead.

Reply:
Oh I see! Thanks!

Reply:
On CI seems to be running cargo run still, can you see why that is?